python/cudf/cudf/core/column/string.py

```diff
@@ -4506,6 +4506,17 @@ def __init__(
             children=children,
         )
 
+    def __sizeof__(self):
+        n = 0
+        if len(self.base_children) == 2:
+            n += (
+                self.base_children[0].__sizeof__()
+                + self.base_children[1].__sizeof__()
+            )
+        if self.base_mask is not None:
+            n += self.base_mask.size
+        return n
```

Comment:
Same here, getting size of `base_mask` and `base_children` instead of the sliced size.